katcp/server.py

```diff
@@ -34,9 +34,12 @@
 from .sampling import format_inform_v5, format_inform_v4
 from .core import (SEC_TO_MS_FAC, MS_TO_SEC_FAC, SEC_TS_KATCP_MAJOR,
                    VERSION_CONNECT_KATCP_MAJOR, DEFAULT_KATCP_MAJOR)
-from .version import VERSION, VERSION_STR
 from .kattypes import (request, return_reply)
 
+# 'import katcp' so that we can use katcp.__version__ later
+# we cannot do this: 'from . import __version__' because __version__
+# does not exist at this stage
+import katcp
```

Comment:
Just to clarify this: `server.py` is imported within `\__init\__.py` before `\__version\__` exists.

The one way around this circular import is to split out `\__version\__` to its own file, i.e. restore `version.py` which we just removed. :-)

The other way is to `import katcp` in full as is done here. A bit brute force but does the job. Revisit if needed.